datastore/integration-test/response/capabilities/200/all_collections.json

```diff
@@ -72,7 +72,7 @@
                 "CoverageJSON"
             ],
             "parameter_names": {
-                "air_pressure_at_sea_level_1_mean_PT1M": {
+                "air_pressure_at_sea_level:1:mean:PT1M": {
```

Comment:
`all_collections` lives in the `capabilities` directory, while `single_collections` lives in the `metadata` directory.
Also, there are no directories like `capabilities.200`, while we have `metadata/200`. Can we reorganize to max consistent? I personally prefer shallower paths, so the first.

I think `all_collections` and `single_collections` are nearly completely the same. Can we get rid of the (500+ lines of) duplication?

Reply:
Made the paths shallow.

Reducing the duplication is possible, but in that case, I think that line `14` of `all_collections.json` should be changed from `data` to `collection`.